app/javascript/src/constants/formInitialValues.js

```diff
@@ -16,5 +16,7 @@ export default {
   notesForm: {
     title: "",
     description: "",
+    contact: "",
```

Comment:
I tried using `contact: {}` since in the end, it is an `object` but, due to some reason Formik is not validating this when it is an object. 
I also tried adding `shape({ label: Yup.string().required(), value: Yup.string().required() })` to the validation schema but I'd no luck with that either. 
In the documentation it is written, "The default cast behavior for object is: `JSON.parse`" so I left it as it is and it works fine.